### ISSUE TITLE
[Docs] fix feast jupyter notebook

### ISSUE THREAD

**Issue by samhita-alla** [flyteorg/flyte#4455]
### Description

the minio URL used in the feast notebook needs to be updated to http://localhost:30002. additionally, the end-to-end example should undergo thorough testing to ensure it is working properly.

### Are you sure this issue hasn't been raised already?

- [X] Yes

### Have you read the Code of Conduct?

- [X] Yes

**Comment by davidmirror-ops** [comment]
For reference: https://github.com/flyteorg/flytesnacks/blob/master/examples/feast_integration/feast_integration/feast_flyte_remote.ipynb

**Comment by manishdait** [comment]
Hi @davidmirror-ops, I would like to work on these issue

**Comment by davidmirror-ops** [comment]
I think it refers to the contents [here](https://github.com/flyteorg/flytesnacks/tree/master/examples/feast_integration/feast_integration)

If I remember correctly, the idea is to align it to the port the minio instance uses on the flyte sandbox, is that correct @samhita-alla ?

**Comment by samhita-alla** [comment]
@davidmirror-ops that's correct. the issue also includes ensuring that the example works by running it on flyte sandbox.

**Comment by manishdait** [comment]
Hey @davidmirror-ops , I have raise the PR for the issue can you please review it, and suggest me changes if any require.
[https://github.com/flyteorg/flytesnacks/pull/1737](Pull  Request)
